packages/aws-cdk-lib/aws-stepfunctions/lib/state-machine.ts

```diff
@@ -407,6 +423,10 @@ export class StateMachine extends StateMachineBase {
    */
   public readonly stateMachineArn: string;
 
+  // Default value for KmsDataKeyReusePeriodSeconds
+  // See: https://docs.aws.amazon.com/step-functions/latest/dg/encryption-at-rest.html#cfn-resources-for-encryption-configuration
```

Comment:
Use JsDoc format

```suggestion
  /** 
   * Default value for `kmsDataKeyReusePeriodSeconds`
   * @see https://docs.aws.amazon.com/step-functions/latest/dg/encryption-at-rest.html#cfn-resources-for-encryption-configuration
   */
```

Additionally, usually keep private and public properties in separate groups, rather than interleaving